code-challenges.js

```diff
@@ -8,62 +8,149 @@
 
 // Pseudo code:
 
-// Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
+// const charCount = (fruit) => {
+//     return
+// }
 
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
-// Expected outcome: "cherry"
+// // Set one:
+// const fruit1 = "apple"
+// const fruit2 = "banana"
+// // Expected outcome: "banana"
+
+// // Set two:
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
+// // Expected outcome: "cherry"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: compare iputs to  212 
+// functions need a return
+// if statement
+// input number 
+//output string
+// let beause boiling point wont change later
+
+// const temperature1 = 42
+
+
+// const temperature2 = 350
+
 
-const temperature1 = 42
-// Expected output: "42 is below boiling point"
+// const temperature3 = 212
 
-const temperature2 = 350
-// Expected output: "350 is above boiling point"
+// const steamBubbles = () => {
+//     if(temp < steamBubbles) 
+//         return temp = "is below boiling point"
+//      else if(temp > steamBubbles)
+//         return temp = "is above boiling point"
+//      else (temp === steamBubbles)
```

Comment:
Remember that when you create a function, you set up your parameters in the parentheses on the first line. Here, you are trying to use `temp`, but it is not set up as a parameter, nor is it defined anywhere as a variable, so the function does not know what `temp` is.
Also, you are comparing `temp` to `steamBubbles`, which is the name of your function. Think about what you want to be doing, what 2 things are you trying to compare for this challenge?